### DISCUSSION TITLE
How to put words between two .. include directives?

### DISCUSSION THREAD

**Discussion by jidanni** [sphinx-doc/sphinx#12005] [Q&A] [answered]
In https://github.com/OSGeo/gdal/pull/9297/commits/164f159131486cd6e1633786fb05ac3df278691d
I am editing
https://gdal.org/programs/gdal_grid.html
i.e.,
https://github.com/OSGeo/gdal/edit/master/doc/source/programs/gdal_grid.rst
and I want to

```
.. include:: options/ot.rst


   put some words here, between these two include statements.


.. include:: options/of.rst
```

but I don't know how.

Sure, I could just put the words inside ot.rst, or even put them into
some ot2.rst, and .. include that, but I don't want to.

[I was told](https://github.com/OSGeo/gdal/pull/9297/commits/164f159131486cd6e1633786fb05ac3df278691d#r1501411945) it wasn't possible.

(Odd, if I paste
`https://github.com/OSGeo/gdal/pull/9297/commits/164f159131486cd6e1633786fb05ac3df278691d#r1501411945`
into this posting, but without the \` protection, I still get a link,
https://github.com/OSGeo/gdal/pull/9297/commits/164f159131486cd6e1633786fb05ac3df278691d#r1501411945
but it no longer shows the comment! OK, I'll report that to GitHub.)

**Comment by jdillard** [comment, marked as answer]
Does removing the indention on the "put some words here, between these two include statements." not produce what you want?

**Reply by jidanni** [reply]
> Does removing the indention on the "put some words here, between these two include statements." not produce what you want?

OK, I'll try it! https://github.com/OSGeo/gdal/pull/9333
